.github/workflows/index.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
           echo "RELEASE_COMMIT=$(git -C llvm-project rev-parse HEAD)" >> $GITHUB_ENV
           echo "RELEASE_COMMIT_SHORT=$(printf '%.12s' $(git -C llvm-project rev-parse HEAD))" >> $GITHUB_ENV
           echo "RELEASE_DATE=$(date -u +%Y%m%d)" >> $GITHUB_ENV
+          echo "RELEASE_NAME=index_$(date -u +%Y%m%d)" >> $GITHUB_ENV
```

Comment:
Or reuse $RELEASE_DATE

Reply:
Unfortunately we can't, as the step that creates/updates the env variable doesn't have access to it, [see](https://docs.github.com/en/enterprise-server@3.4/actions/using-workflows/workflow-commands-for-github-actions#:~:text=The%20step%20that%20creates%20or%20updates%20the%20environment%20variable%20does%20not%20have%20access%20to%20the%20new%20value%2C%20but%20all%20subsequent%20steps%20in%20a%20job%20will%20have%20access.)